clients/include/testing_axpy.hpp

```diff
@@ -84,10 +84,15 @@ rocblas_status testing_axpy(Arguments argus)
     rocblas_int N = argus.N;
     rocblas_int incx = argus.incx;
     rocblas_int incy = argus.incy;
-    T h_alpha = argus.alpha;
     rocblas_int safe_size = 100;   // arbitrarily set to 100
-
-    std::unique_ptr<rocblas_test::handle_struct> test_handle(new rocblas_test::handle_struct);
+    T h_alpha = 0;
```

Comment:
I think this needs to change to     T h_alpha = argus.alpha;

Reply:
It's initialized just below, depending on T type